tests/compose/test-basic-unified.sh

```diff
@@ -75,6 +75,16 @@ ostree --repo="${repo}" cat "${treeref}" /usr/lib/tmpfiles.d/pkg-pam.conf > auto
 assert_file_has_content_literal autovar.txt 'd /run/console'
 echo "ok autovar"
 
+# Check symlinks injected into the rootfs.
+ostree --repo="${repo}" ls "${treeref}" /usr/lib/alternatives /usr/lib/vagrant | grep '^d00755'> content.txt
+assert_file_has_content_literal content.txt '/usr/lib/alternatives'
+assert_file_has_content_literal content.txt '/usr/lib/vagrant'
+ostree --repo="${repo}" ls "${treeref}" /var/lib/alternatives /var/lib/vagrant /usr/local | grep '^l00777' > content.txt
+assert_file_has_content_literal content.txt '/usr/local -> ../var/usrlocal'
+assert_file_has_content_literal content.txt '/var/lib/alternatives -> ../../usr/lib/alternatives'
+assert_file_has_content_literal content.txt '/var/lib/vagrant -> ../../usr/lib/vagrant'
+echo "ok symlinks"
```

Comment:
There is a shared `basic_test` where we put everything in common to test between the unified and non-unfied paths. This looks like it belongs there instead?

Reply:
Oh indeed, I didn't realize that and manually duplicated it instead. Amended, unified now.